include/openssl/rsa.h

```diff
@@ -473,7 +473,7 @@ OPENSSL_EXPORT int RSA_check_key(const RSA *rsa);
 // RSA_check_fips performs public key validity tests on |key|. It returns one if
 // they pass and zero otherwise. Opaque keys always fail. This function does not
 // mutate |rsa| for thread-safety purposes and may be used concurrently.
-OPENSSL_EXPORT int RSA_check_fips(RSA *key);
+OPENSSL_EXPORT int RSA_check_fips(const RSA *key);
```

Comment:
Based upon the assertion made in the documentation here -- "_This function does not mutate |rsa| and for thread-safety purposes may be used concurrently_" -- shouldn't the parameter be `const`?  (If our implementation calls a mutating function, then we should remove this claim from the documentation.)